db/migrate/20230117190802_create_workflow_migrations.rb

```diff
@@ -0,0 +1,26 @@
+class CreateWorkflowMigrations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflows do |t|
+      t.references :ems,           :type => :bigint, :index => true, :references => :ext_management_system
+      t.references :tenant,        :type => :bigint, :index => true
+      t.references :git_reference, :type => :bigint, :index => true
+      t.string :type, :index => true
+      t.jsonb :payload, :default => {}
```

Comment:
Not sure about the name payload, but can't come up with anything better.

Reply:
yeah I was going to call it `t.jsonb :workflow` but `workflow.workflow` felt repetitive repetitive

Reply:
Is payload the starting workflow content or the starting context content? Wonder if the word `initial_?` would be useful here.

Reply:
Payload is the actual ASL JSON

Reply:
I'm ok with keeping payload, but wondering if `initial_workflow`, `workflow_content`, `initial_workflow_content`, or something like that would be clearer.

Reply:
So the only reason I don't like `initial_*` is because the actual workflow json payload doesn't change during the execution.  I agree I don't love `payload` so no issues with changing the name.

Reply:
I'll change it to `workflow_content` and we can throw ideas around if there is a better name than that.